src/cljs/replumb/repl.cljs

```diff
@@ -759,11 +779,11 @@
   ([]
    (reset-env! nil))
   ([namespaces]
+   (read-eval-call {} identity "(in-ns 'cljs.user)")
```

Comment:
Here we need to call `in-ns 'cljs.user` because if we call it after `purge-ns` `current-ns` of `@app-env` will be `nil` (!) ..
hence the `nil` in the known-namespaces